checklists/alz_checklist.en.json

```diff
@@ -1220,127 +1183,309 @@
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "346840b8-1064-496e-8396-4b1340172d52",
+            "id": "D07.15",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/premium-features#tls-inspection"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Consider Web Categories to allow or deny outbound access in bulk.",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "39990a13-915c-45f9-a2d3-562d7d6c4b7c",
+            "id": "D07.16",
+            "severity": "Low",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/premium-features#web-categories"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Use Azure Firewall in parallel/sequence with Azure App Gateway",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "6eff7e6c-6c4a-43d7-be3f-6641c2cb3d4a",
+            "id": "D07.17",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/architecture/example-scenario/gateway/firewall-application-gateway#firewall-and-application-gateway-in-parallel"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Enable Azure Firewall DNS proxy configuration ",
+            "waf": "Security",
+            "service": "Firewall",
+            "guid": "94f3eede-9aa3-4088-92a3-bb9a56509fad",
+            "id": "D07.18",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/dns-details"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Restrict usage of Public IP addresses directly tied to Virtual Machines",
+            "waf": "Security",
+            "service": "Firewall",
+            "guid": "3c5a808d-c695-4c14-a63c-c7ab7a510e41",
+            "id": "D07.19",
+            "severity": "Medium",
+            "link": "https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Policies#corp"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Integrate Azure Firewall with Azure Monitor",
+            "waf": "Monitor",
+            "service": "Firewall",
+            "guid": "1dc04554-dece-4ffb-a49e-5c683e09f8da",
+            "id": "D07.20",
+            "severity": "Low",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/firewall-diagnostics"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
```

Comment:
This is not firewall specific, and but I am seeing that we don't have a general Defender for Cloud recommendation.  This should go in another section, though.

Reply:
I have removed this now

Reply:
This is still showing up in the version here - is there a commit that needs to be made for it?